ydb/docs/en/core/dev/example-app/java/index.md

```diff
@@ -36,116 +35,85 @@ App code snippet for driver initialization:
-        .setPrimaryKeys("series_id", "season_id", "episode_id")
-        .build();
-
-    retryCtx.supplyStatus(session -> session.createTable(database + "/episodes", episodesTable))
-            .join().expectSuccess("Can't create table /episodes");
+    retryCtx.supplyResult(session -> session.createQuery(""
+            + "CREATE TABLE series ("
+            + "  series_id UInt64,"
+            + "  title Text,"
+            + "  series_info Text,"
+            + "  release_date Date,"
+            + "  PRIMARY KEY(series_id)"
+            + ")", TxMode.NONE).execute()
+    ).join().getStatus().expectSuccess("Can't create table series");
+
+    retryCtx.supplyResult(session -> session.createQuery(""
+            + "CREATE TABLE seasons ("
+            + "  series_id UInt64,"
+            + "  season_id UInt64,"
+            + "  title Text,"
+            + "  first_aired Date,"
+            + "  last_aired Date,"
+            + "  PRIMARY KEY(series_id, season_id)"
+            + ")", TxMode.NONE).execute()
+    ).join().getStatus().expectSuccess("Can't create table seasons");
+
+    retryCtx.supplyResult(session -> session.createQuery(""
+            + "CREATE TABLE episodes ("
+            + "  series_id UInt64,"
+            + "  season_id UInt64,"
+            + "  episode_id UInt64,"
+            + "  title Text,"
+            + "  air_date Date,"
+            + "  PRIMARY KEY(series_id, season_id, episode_id)"
+            + ")", TxMode.NONE).execute()
+    ).join().getStatus().expectSuccess("Can't create table episodes");
 }
 ```
 
-You can use the `Session.describeTable()` method to view information about the table structure and make sure that it was properly created:
-
-```java
-private void describeTables() {
-    logger.info("--[ DescribeTables ]--");
-
-    Arrays.asList("series", "seasons", "episodes").forEach(tableName -> {
-        String tablePath = database + '/' + tableName;
-        TableDescription tableDesc = retryCtx.supplyResult(session -> session.describeTable(tablePath))
-                .join().getValue();
-
-        List<String> primaryKeys = tableDesc.getPrimaryKeys();
-        logger.info(" table {}", tableName);
-        for (TableColumn column : tableDesc.getColumns()) {
-            boolean isPrimary = primaryKeys.contains(column.getName());
-            logger.info("     {}: {} {}", column.getName(), column.getType(), isPrimary ? " (PK)" : "");
-        }
-    });
-}
-```
 {% include [../steps/03_write_queries.md](../_includes/steps/03_write_queries.md) %}
 
-Code snippet for data insert/update:
+To execute YQL queries, use the `QuerySession.createQuery()` method. It creates a new `QueryStream` object, which allows to execute a query and subscribe for receiving response data from the server. Because the write requests don't expect any results, the `QueryStream.execute()` method is used without parameters; it just executes the request and waits for the stream to complete.
```

Comment:
```suggestion
To execute YQL queries, use the `QuerySession.createQuery()` method. It creates a new `QueryStream` object, which allows to execute a query and subscribe for receiving response data from the server. Because write requests don't expect any results, the `QueryStream.execute()` method is used without parameters; it just executes the request and waits for the stream to complete.
```